erpnext/hr/doctype/expense_claim/expense_claim.py

```diff
@@ -405,3 +405,28 @@ def create_expense_claim(user, expense_date, expense_claim_type, description, am
 			}
 		]
 	}).insert(ignore_permissions=True, ignore_mandatory=True)
+
+
+@frappe.whitelist()
+def list_expense_claims(user):
+	"""
+	API endpoint to get all Expense Claim list and its child table
+
+	User email id args required to list existing data created by that perticular user (Request From Driver App).
+	"""
+
+	employee = frappe.get_value("Employee", {"company_email": user}, "name" )
+	if not employee:
+		return __("Employee's E-mail ID Not Found, Please Add 'Company Email' Under 'Employee' Document")
+
+	expense_claims = frappe.get_all("Expense Claim", filters={"employee": employee})
+
+	expense_claim_list = []
+
+	if not expense_claims:
+		return "No Claim Found For "+ employee
+
+	for claim in expense_claims:
+		expense_claim_list.append(frappe.get_doc("Expense Claim", claim))
```

Comment:
You do not need to do a `get_doc` since you can get all the fields in `get_all` itself